docker/Dockerfile

```diff
@@ -1,22 +1,25 @@
-FROM debian:jessie
+FROM frolvlad/alpine-glibc
```

Comment:
Yes it's required. Took me a while to figure it out. 
alpine comes with musl libc. Both the manager executeable and the factorio headless server seem to be compiled against glibc though. This will cause linux to error out with the message "File does not exist" when you try to run any of these two binaries.